Sources/SwiftGraphQLClient/Client/Selection.swift

```diff
@@ -90,6 +90,17 @@ extension GraphQLClient {
             .tryMap { result in try result.decode(selection: selection) }
             .eraseToAnyPublisher()
     }
+
+
+    /// Executes a query request with given execution parameters.
+    public func query<T, TypeLock>(
```

Comment:
These methods are tested, but tests feel super awkward 😨 Would be awesome if you could help make them more manageable 🙇

Reply:
I'd add a new view file to the DEMO app and manually test that they work as expected. 

Maybe you could add an extra comment (not documentation) to their declarations saying "Test in the demo application by opening XYZ".

Reply:
Oh, that's a good idea